src/HL7/Message.php

```diff
@@ -87,7 +87,7 @@ public function __construct(
         $this->subcomponentSeparator = $hl7Globals['SUBCOMPONENT_SEPARATOR'] ?? '&';
         $this->repetitionSeparator = $hl7Globals['REPETITION_SEPARATOR'] ?? '~';
         $this->escapeChar = $hl7Globals['ESCAPE_CHAR'] ?? '\\';
-        $this->hl7Version = $hl7Globals['HL7_VERSION'] ?? '2.3';
+        $this->hl7Version = $hl7Globals['HL7_VERSION'] ?? '2.5.1';
```

Comment:
Hmmm... 2.5..1 is backward compatible.

Reply:
Have a look at the HL7 history.
Version 2.3 is more than 23 years old:
https://lyniate.com/wp-content/uploads/image-5.webp

Btw, in fact, it doesn't matter if the segments are more recent.

Reply:
But...
Keeping the default version to 2.3 is inconsistant, because some methods changed their names from 2.5.
In the case you really want to keep it to 2.3 by default, some method aliases should be added to ensure the users of older version will not get issues.

Reply:
We really need to keep 2.3 for now, as otherwise all users of this library have to update their Message() constructor. We're still not ready for this breaking change.

We can support 2.5.1 provided 2.3 continues to work as it is, at least for now until a full upgrade plan is prepared.